src/types/deliveryMode.ts

```diff
@@ -1,6 +1,6 @@
 import DeliveryModes from 'constants/deliveryModes';
 
-interface DeliveryModeSubmitValues {
+export interface DeliveryModeSubmitValues {
```

Comment:
If you export it as named export, remove default export. Check `paymentMethod` as well